Cargo.toml

```diff
@@ -43,7 +43,10 @@ tokio-rustls = "0.26.1"
 rustls = "0.23.21"
 rcgen = "0.11"
 time = "0.3"
-
+ctor = "0.2"
+syn = { version = "2.0", features = ["full"] }
+quote = "1.0"
```

Comment:
Does this dependency need to be here since it's in the `redis-test-macro` crate?